shared/hooks/useSearch.ts

```diff
@@ -49,18 +67,22 @@ const useSearch = (documents: SearchResult[], query: string): SearchResult[] =>
   }, [query])
 
   useEffect(() => {
-    if (!indexing && search.documentCount !== documents.length) {
+    const sanitizedDocuments = removeDuplicatedPaths(documents)
```

Comment:
After replacing it with:
```js 
const sanitizedDocuments = removeDuplicatedPaths([...documents, documents[0]!])
```
It throws the following:
`Cannot read properties of undefined (reading 'path')`
and highlights this line as the source of the issue: 
```js
 const isNew = !paths.has(document.path)
```

Reply:
I guess its better to follow @LeandraH's testing instructions [here](https://github.com/digitalfabrik/integreat-app/pull/3200#pullrequestreview-2768038950) as this is the original issue (and don't forget to set the cms url back to the test tms afterwards). However, if you would still like to test it this way, you can try this:
```ts
    const sanitizedDocuments = removeDuplicatedPaths(documents.length > 0 ? [...documents, documents[0]!] : [])
```